test/Microsoft.AspNetCore.Cors.Test/CorsPolicyBuilderTests.cs

```diff
@@ -299,5 +299,98 @@ public void DisallowCredential_SetsSupportsCredentials_ToFalse()
             var corsPolicy = builder.Build();
             Assert.False(corsPolicy.SupportsCredentials);
         }
+
+        [Fact]
+        public void GetNormalizedOrigin_ReturnsLowerCasedValue_IfStringIsNotParseableAsUrl()
```

Comment:
If the string is not parsable as a URI then the casing is the least of their worries.

Reply:
I sort of wanted to make sure the behavior is no different (we don't throw for instance) from what it looks like today.